ukvm/ukvm-core.c

```diff
@@ -294,8 +294,18 @@ static void load_code(const char *file, uint8_t *mem,     /* IN */
 
 static void setup_system_64bit(struct kvm_sregs *sregs)
 {
-    sregs->cr0 |= X86_CR0_PE;
-    sregs->efer |= EFER_LME;
+    uint64_t cr0 = (X86_CR0_NE | X86_CR0_PE | X86_CR0_PG)
+        & ~(X86_CR0_NW | X86_CR0_CD);
+    uint64_t cr4 = X86_CR4_PAE;
```

Comment:
cr4:`X86_CR4_PAE`  and cr0:`X86_CR0_PG` are already set in `setup_system_page_tables`. You might want to remove them from `setup_system_page_tables` or here.